src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/control/AbstractTankTab.java

```diff
@@ -1,14 +1,155 @@
+     * @param tank The AbstractTank which is controlled through the AbstractTankTab.
+     */
+    public AbstractTankTab(String name, AbstractTank tank) {
+        super(name);
+        this.setClosable(false);
+
+        pane = new GridPane();
+        pane.setPrefWidth(ViewConstants.CONTROL_WIDTH);
+        row = 0;
+
+        createTemperatureSlider(tank);
+        createOutFlowSlider(pane, tank);
+    }
+
+    private void createTemperatureSlider(AbstractTank tank) {
+        Translator t = Translator.getInstance();
+        int col = 0;
+
+        Label temperatureLabel = new Label(t.getString("simulation.view.temperatureLabel"));
+        pane.add(temperatureLabel, col++, row);
+
+        // Otherwise the scale might end at -0.0
+        float min = SimulationConstants.MIN_TEMPERATURE - SimulationConstants.CELCIUS_OFFSET;
+        if (min == -0.0) {
+            min = 0.0f;
+        }
+
+        temperatureSlider = new Slider(min,
+                SimulationConstants.MAX_TEMPERATURE - SimulationConstants.CELCIUS_OFFSET,
+                tank.getLiquid().getTemperature() - SimulationConstants.CELCIUS_OFFSET);
+        temperatureSlider.setShowTickLabels(true);
+        temperatureSlider.setShowTickMarks(true);
+        temperatureSlider.setMajorTickUnit(25);
+        temperatureSlider.setMinorTickCount(9);
+        temperatureSlider.setPrefWidth(ViewConstants.CONTROL_SLIDER_WIDTH);
+        GridPane.setHgrow(temperatureSlider, Priority.ALWAYS);
+        pane.add(temperatureSlider, col++, row);
+        GridPane.setMargin(temperatureSlider, ViewConstants.CONTROL_PADDING);
+
+        //Labels to show the current value and unit
+        TextField temperatureValue = new TextField("" + tank.getLiquid().getTemperature());
+        temperatureValue.setMaxWidth(ViewConstants.CONTROL_INPUT_WIDTH);
+        pane.add(temperatureValue, col++, row);
+        GridPane.setMargin(temperatureValue, ViewConstants.CONTROL_PADDING);
+        Label temperatureSign = new Label(t.getString("simulation.view.temperatureUnit"));
+        pane.add(temperatureSign, col++, row);
+        GridPane.setMargin(temperatureSign, ViewConstants.CONTROL_PADDING);
+
+        StringProperty sp = temperatureValue.textProperty();
+        DoubleProperty dp = temperatureSlider.valueProperty();
+        StringConverter<Number> converter = new NumberStringConverter();
+        Bindings.bindBidirectional(sp, dp, converter);
+        row++;
+    }
+
+    private void createOutFlowSlider(GridPane pane, AbstractTank tank) {
+        Translator t = Translator.getInstance();
+        int col = 0;
+
+        Label outFlowLabel = new Label(t.getString("simulation.view.outFlowLabel"));
+        pane.add(outFlowLabel, col++, row);
+
+        outFlowSlider = new Slider(0, 100, tank.getOutPipe().getValveThreshold());
+        outFlowSlider.setShowTickLabels(true);
+        outFlowSlider.setShowTickMarks(true);
+        outFlowSlider.setMajorTickUnit(10);
```

Comment:
https://github.com/ByteHamster/PSE/blob/master/src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/MonitoringViewConstants.java#L61

In der Simulation nicht, aber in der Überwachungskonsole.